fix: improve readme (#103)
graasp/graasp-web: README.md

### PostgreSQL

The admin platform uses the same database as the core platform.

We recommend that you use the database provided by the devcontainer for the graasp/core project when running the admin.

> For tests and in case you would only need access to things that the admin does (nothing related to graasp) you would be fine using a local postgres instance for example via a docker container, or with an app such as [Postgres.app](https://postgresapp.com/) on MacOS.

Reviewer: ```suggestion
For tests or if you only need to access admin's related data, you will be fine using a local postgres instance for example via a docker container, or with an app such as [Postgres.app](https://postgresapp.com/) on MacOS.
```

#### Devcontainer database

> You want to use all features of graasp and have an install of the core project running in the devcotnainer.

Reviewer: ```suggestion
You want to use all features of graasp and have an install of the core project running in the devcontainer.
```
